pages/index.js

```diff
@@ -47,8 +48,9 @@ export default function Home({ date, expirationDate }) {
       )}
       <WarningText>
         Please note that applications for the Discretionary Business Grant close
-        at 23:59 on Friday 28th August 2020. Any applications submitted after
-        this time will be invalid.
+        at {dateFormat(new Date(expirationDate), 'h:MM:ss TT')} on{' '}
+        {dateFormat(new Date(expirationDate), 'dddd, mmmm dS, yyyy')}. Any
```

Comment:
I think following the GOV.UK guidelines for writing out dates is good practice; https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#dates

```suggestion
        {dateFormat(new Date(expirationDate), 'dddd, mmmm dS yyyy')}. Any
```